score-vmware/src/main/java/io/cloudslang/content/vmware/constants/ErrorMessages.java

```diff
@@ -35,6 +35,10 @@
     public static final String ATAPI_CONTROLLER_CAPACITY_MAXED_OUT = "The IDE controller on the vm has maxed out its " +
             "capacity. Please add an additional IDE controller";
 
+    public static final String NOT_BYTE = "The input value must be a byte number.";
     public static final String NOT_INTEGER = "The input value must be a int number.";
     public static final String NOT_LONG = "The input value must be a long number.";
```

Comment:
"The input value must be a long number"... does the user know the range of long? Do negative numbers count as valid values ? 

I suggest changing the messages and explicitly say what the maximum value is for each.